core/model/smarty/Smarty.class.php

```diff
@@ -27,7 +27,7 @@
  * @author    Uwe Tews   <uwe dot tews at gmail dot com>
  * @author    Rodney Rehm
  * @package   Smarty
- * @version   3.1.33
+ * @version   3.1.34-dev
```

Comment:
Someone forgot this one ;)

Reply:
https://github.com/modxcms/revolution/pull/15249/files#diff-63e18788246239669f3b7080afe2bb95R115

Reply:
=)